### PR TITLE
Bluebird changes & PCM 1525 - showing external tracker updates in case discussion section

### PR DESCRIPTION
**Pull request by gandhikeyur**
@vrathee @renujhamtani Please review

### CODE REVIEW

**Comment by renujhamtani** [comment]
Looks Good!

**Comment by vrathee** [comment]
Reviewed !

File: app/cases/views/discussionSection.jade [hunk 1]
```diff
@@ -83,6 +83,17 @@ ul#tab_list.nav.nav-tabs(role="tablist")
               span {{'at'|translate}}&nbsp;
               time {{element.last_modified_time}}
             .pre-wrap(ng-if="element.comment_type === 'chat'", ng-bind-html="parseCommentHtml(element)")
+          article.comment.article-external-updates(ng-if="element.originating_system")
+              i.fa.fa-comments-o
+              h4.meta {{'External Tracker Updates'|translate}} - {{element.originating_system}}
```
Comment (vrathee) [inline app/cases/views/discussionSection.jade]: Use a single date filter instead of splitting in service.

Comment (gandhikeyur) [inline app/cases/views/discussionSection.jade]: @vrathee Can not use single date filter as the current version of angular(1.2.1) does not support timezone filtering(formatting of 'Z'), we want timezone to be displayed as +05:30 and date filter shows this as +0530. This formatting is available from angular version 1.3.0. We can use this filter in PCM once we upgrade our angular version.

**Pull request merged**